conda-recipe/conda_build_config.yaml

```diff
@@ -15,4 +15,8 @@
 #===============================================================================
 
 numpy:
-  - 1.19
+  - 1.24
+c_compiler:    # [win]
+- vs2022       # [win]

```

Comment:
Private CI Visual Studio doesn't utilize this compiler, though public CI does. I think it worth having a discussion how we verify this support on private CI-side thoughts? @homksei @ethanglaser

Reply:
I don't see any need to re-specification of `vs` version until we are enable building of conda package in internal CI: azure CI images and most of users use vs2022 anyway

Reply:
Seems reasonable.  I will still poke at private CI people as to why we are using an outdated vs version since most are using 2022 as you mentioned.